python/cudf/cudf/tests/test_series.py

```diff
@@ -462,7 +465,8 @@ def test_series_describe_other_types(ps):
     gs = cudf.from_pandas(ps)
 
     expected = ps.describe()
```

Comment:
There will be no warnings on pandas end for this because it was a directly removed parameter without any deprecations.